.travis.yml

```diff
@@ -130,22 +130,6 @@ matrix:
       name: 'Backend [Postgres] (2/2)'
       env: TEST_SUITE=postgres DB=postgres TOTAL_TEST_GROUPS=2 TEST_GROUP=1
 
-    # django 1.8 compatibility
-    - <<: *postgres_default
-      name: 'Backend [Postgres] (Django 1.8) (1/2)'
-      env: TEST_SUITE=postgres DB=postgres DJANGO_VERSION=">=1.8,<1.9" TOTAL_TEST_GROUPS=2 TEST_GROUP=0
-    - <<: *postgres_default
-      name: 'Backend [Postgres] (Django 1.8) (2/2)'
-      env: TEST_SUITE=postgres DB=postgres DJANGO_VERSION=">=1.8,<1.9" TOTAL_TEST_GROUPS=2 TEST_GROUP=1
-
-    # django 1.8 compatibility with migrations
```

Comment:
Do we need to keep this `SOUTH_TESTS_MIGRATE=0` test in place?

Reply:
I can switch up the tests to run without migrations as with them they are a fair bit slower.